tests/fault_tolerance/cancellation/utils.py

```diff
@@ -261,11 +268,11 @@ def send_cancellable_request(
         prompt += " Make sure it is" + " long" * 16000 + "!"
 
     if request_type == "completion":
-        return send_completion_request(prompt, 16384)
+        return send_completion_request(prompt, 8192, frontend_port)
     elif request_type == "chat_completion":
-        return send_chat_completion_request(prompt, 16384, stream=False)
+        return send_chat_completion_request(prompt, 8192, frontend_port, stream=False)
```

Comment:
Please don't change the prompt length, they are increased for faster machines.
```suggestion
        return send_chat_completion_request(prompt, 16384, frontend_port, stream=False)
```

Reply:
changing back